docs/alerts.rst

```diff
@@ -48,6 +48,15 @@ for more complex searches. The example below will search with a solr query searc
     9d327888-<truncated> WINDOWS WINDOWS-TEST THREAT
     aab3c640-<truncated> WINDOWS WINDOWS-TEST THREAT
 
+.. tip::
+    When filtering by fields that take a list parameter, an empty list will be treated as a wildcard and match everything.
+
+Ex: Returns all types
+
+.. code-block:: python
+
+    >>> alerts = (cb.select(BaseAlert).set_types([]))
```

Comment:
Can make this `list()` instead of using the parantheses shortcut

Reply:
done